Cargo.toml

```diff
@@ -64,7 +64,7 @@ locales = ["en", "fr"]
 namespaces = ["nav"]
 
 [features]
-default = ["csr"]
+default = ["ssr"]
```

Comment:
Why make SSR the default?

Reply:
csr feature is not a target for us now. ssr is findamental